### PR TITLE
Renaming tests from origin- to openshift- and other naming updates

### LINKED ISSUE #16: Update names
Perhaps we want `smoke-test.sh` vs `protractor.sh`, to make it easier for non-JS developers to be able to find the entrypoints, once debugging is needed.

### PR DESCRIPTION
**Pull request by jhadvig**
CHANGES:
- protractor.sh  ->  smoke-test.sh
- TEST_INTERVAL  -> TEST_INTERVAL_MINUTES
- updated the `smoke-test-list.yaml` to contain a Service co secure comunication

Fixes #16 

@benjaminapetersen PTAL

### CODE REVIEW

File: kube/deployments/smoke-test.yaml [hunk 1]
```diff
@@ -15,20 +15,19 @@ spec:
...
         env:
         - name: CONSOLE_URL
-          value: https://10.192.213.145:8443
-        - name: CONSOLE_USER
-          value: bob
-        - name: CONSOLE_PASSWORD
-          value: bob
+          value: https://192.168.1.69:8443
+        - name: TEST_INTERVAL_MINUTES
+          value: '2'
```
Comment (benjaminapetersen) [inline kube/deployments/smoke-test.yaml]: Did we say 2 mins, or 5 mins for default?

Comment (jhadvig) [reply kube/deployments/smoke-test.yaml]: default is 5 but I've set the variable here for demo purposes :)

File: test/environment.js [hunk 2]
```diff
@@ -22,7 +22,7 @@ const auth = {
 };
 
 // Default test interval is 5 minutes
```
Comment (benjaminapetersen) [inline test/environment.js]: In this comment series https://github.com/openshift/openshift-ansible/pull/7880/files#r180740741 it was suggested we accept a `time.Duration` string.  That probably makes sense, with a check for a min value.

Comment (benjaminapetersen) [reply test/environment.js]: I think that means something like this:https://golang.org/pkg/time/#example_Duration

    func ParseDuration

    func ParseDuration(s string) (Duration, error)
    ParseDuration parses a duration string. A duration string is a possibly signed sequence of decimal numbers, 
    each with optional fraction and a unit suffix, such as "300ms", "-1.5h" or "2h45m". Valid time units are "ns", "us" 
    (or "µs"), "ms", "s", "m", "h".




I bet that this npm package might cover it.  https://www.npmjs.com/package/parse-duration

We could spin off as a separate issue if you want to merge this PR.

Comment (jhadvig) [reply test/environment.js]: Lets do that as followUp :)

Comment (benjaminapetersen) [reply test/environment.js]: created #27 for this

**Pull request merged**